.github/workflows/tests.yml

```diff
@@ -67,18 +77,23 @@ jobs:
           path: bazel-testlogs
 
   lts_ios_integration_tests:
-    name: Build and Test ( Virtual Frameworks + LTS )
+    name: Bazel 5 - Build and Test (Virtual Frameworks)
```

Comment:
If we're going to be updating these, I'd prefer it stay beside the `Virtual Frameworks` portion since it's the "modifiers" to the build. I think the prefix makes the build list hard to parse:
<img width="578" alt="Screenshot 2023-05-10 at 9 29 18 AM" src="https://github.com/bazel-ios/rules_ios/assets/5728070/a15143dd-a418-4207-8fc4-8b2de766ace3">
So I guess:
```suggestion
    name: Build and Test (Virtual Frameworks, Bazel 5)
```

Reply:
That makes sense, went back and forth on it